buildconfig/Jenkins/Conda/conda-buildscript

```diff
@@ -40,10 +41,24 @@ WORKSPACE=$1
 shift
 CMAKE_PRESET=$1
 shift
+
 EXPECTED_MAMBAFORGE_PATH=$WORKSPACE/mambaforge # Install into the WORKSPACE_DIR
-EXPECTED_CONDA_PATH=$EXPECTED_MAMBAFORGE_PATH/bin/conda
+if [[ $OSTYPE == "msys" ]]; then
+    EXPECTED_CONDA_PATH=$EXPECTED_MAMBAFORGE_PATH/condabin/mamba.bat
+else
+    EXPECTED_CONDA_PATH=$EXPECTED_MAMBAFORGE_PATH/bin/mamba
+fi
```

Comment:
A lot of this setup looks common, could we extract the common elements into a separate file at all, or is it more effort than it's worth?